### ISSUE TITLE
[HOW-TO] [UCI] Open related record from list(Related - Common)

### ISSUE THREAD

**Issue by saurabhchanchawat** [Microsoft/EasyRepro#824]
I wanted to select 'Alerts' or 'Income' or any other related menu from list.
I am using `xrmApp.Entity.SelectTab("Related", "Income");` to perform the action. But, it is not working and giving exception as 'The tab with name: Related does not exist'. 
Is there is anything which is wrong here?
![image](https://user-images.githubusercontent.com/23567469/79633373-bde73800-8182-11ea-9620-6d49f8c231db.png)

**Comment by saurabhchanchawat** [comment]
After dig into the code, I see in "ClickTab" method, at last else condition, code is throwing exception. That's why it is not clicking on Related record
```
internal void ClickTab(IWebElement tabList, string xpath, string name)
        {
            IWebElement moreTabsButton;
            IWebElement listItem;
            // Look for the tab in the tab list, else in the more tabs menu
            IWebElement searchScope = null;
            if (tabList.HasElement(By.XPath(string.Format(xpath, name))))
            {
                searchScope = tabList;
            }
            else if (tabList.TryFindElement(By.XPath(AppElements.Xpath[AppReference.Entity.MoreTabs]), out moreTabsButton))
            {
                moreTabsButton.Click();
                searchScope = Browser.Driver.FindElement(By.XPath(AppElements.Xpath[AppReference.Entity.MoreTabsMenu]));
            }
            if (searchScope.TryFindElement(By.XPath(string.Format(xpath, name)), out listItem))
            {
                listItem.Click(true);
            }
            else
            {
               //throw new Exception($"The tab with name: {name} does not exist");
            }
        }

```
When I commented this line, it is working perfectly.

**Comment by TYLEROL** [comment]
@saurabhchanchawat - Thanks for raising this. We've merged a fix for SelectTab in #859. With some additional changes going in later today with #864 

Once these are merged, we plan to update the release branch and NuGet packages.

Thanks,
Tyler
